docs/platforms/python/configuration/filtering/index.mdx

```diff
@@ -20,7 +20,7 @@ The `before_send` callback allows you to modify the event payload before the SDK
 
 The callback receives two arguments: the event payload and a [hint](#event-hints). The callback can return either the event payload to send to Sentry or `None` if the event should be dropped.
 
-Since the SDK sends to Sentry the exact payload that the callback returns, the callback can perform arbitrary modifications to the event payload, including adding data to the event and modifying or deleting existing event fields. It is even possible to use `before_send` to construct a completely different event payload from scratch.
+Since the SDK sends to Sentry the exact payload that the callback returns, the callback can perform arbitrary modifications to the event payload, including adding data to the event and modifying or deleting existing event fields.
```

Comment:
```suggestion
Since the SDK sends the same payload to Sentry that the callback returns, the callback can perform arbitrary modifications to the event payload, including adding data to the event and modifying or deleting existing event fields.
```